book/src/bridge/traits.md

```diff
@@ -26,3 +26,7 @@ For further documentation, refer to the documentation of the individual traits:
 - [Constructor](https://docs.rs/cxx-qt/latest/cxx_qt/trait.Constructor.html) - custom constructor
 - [Initialize](https://docs.rs/cxx-qt/latest/cxx_qt/trait.Initialize.html) - execute Rust code when the object is constructed
 - [Threading](https://docs.rs/cxx-qt/latest/cxx_qt/trait.Threading.html) - marker trait whether CXX-Qt threading should be enabled
+- `Upcast` - Allows a type to access its parent class if there is one
+  - ⚠️ Should not be implemented manually by the user
```

Comment:
In theory for extern C++Qt / RustQt types yes the user/developer should not implement. However if you are building your own bindings like cxx-qt-lib you may need to manually implement them ?

Unsure how we communicate / phrase this :thinking: